src/auth/auth.js

```diff
@@ -188,7 +199,7 @@ Auth.prototype.resumeSession = async function() {
 		}
 	}
 	if (!success) {
-		localStorage.removeItem(TokenName)
+		sessionStorage.removeItem(TokenName)
```

Comment:
could we have functions instead for everything which calls sessionStorage. for example in this case Auth.prototype.clearToken()

Reply:
true. replaced with call to this.clearToken